src/commands/cmd_tdigest.cc

```diff
@@ -131,6 +131,44 @@ class CommandTDigestInfo : public Commander {
   std::string key_name_;
 };
 
+class CommandTDigestAdd : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    key_name_ = args[1];
+
+    values_.reserve(args.size() - 2);
+    for (size_t i = 2; i < args.size(); i++) {
+      auto value = ParseFloat(args[i]);
+      if (!value) {
+        return {Status::RedisParseErr, errValueIsNotFloat};
```

Comment:
I test this case in redis stack latest image, the error message is `error parsing val parameter`.

Maybe we could follow current redis's error message if possible.😊